dtm-database-backend/src/main.rs

```diff
@@ -1,3 +1,13 @@
-fn main() {
-  println!("Hello, world!");
+use axum::{routing::get, Router};
+
+#[tokio::main]
+async fn main() {
+  // build our application with a single route
+  let app = Router::new().route("/", get(|| async { "Hello, World!" }));
+
+  // run it with hyper on localhost:3000
+  axum::Server::bind(&"0.0.0.0:3000".parse().unwrap())
```

Comment:
ポート番号は、
1. 環境変数`PORT`を見る
2. `$PORT`がN/Aなら`3001`

という感じで決定して欲しいです！！

`"0.0.0.0:3000"`みたいになんでもかんでも文字列に押しこむのも……なので、
この書き方がいい感じです:
```suggestion
  let address = SocketAddr::from((Ipv4Addr::UNSPECIFIED, /*ポート*/));
  axum::Server::bind(&address.parse().unwrap())
```